client/src/pages/Country/Country.tsx

```diff
@@ -264,11 +265,11 @@ export const Country: React.FC<CountryProps> = ({ country }) => {
                 )
                 .map((video) => ({
                   x: video.pub_date,
-                  y: video.pub_to_trend,
+                  y: parseInt(video.pub_to_trend),
                   custom: video.video_id.length < 1,
                 }))}
-                title = {"Experimental"}
-              color
+                size = {genResults.length}
```

Comment:
In this case the length is actually genResults + expResults and not genResults, although expResults is negligible in length comparatively